docs/core/extensions/logging.md

```diff
@@ -169,6 +226,34 @@ On [Azure App Service](https://azure.microsoft.com/services/app-service/), selec
 
 For more information on setting .NET configuration values using environment variables, see [environment variables](configuration-providers.md#environment-variable-configuration-provider).
 
+### Configure logging with code
+
+To configure logging in code, use the <xref:Microsoft.Extensions.Logging.ILoggingBuilder> API. This can be accessed from different places:
+
+- When creating the `ILoggerFactory` directly, configure in <xref:Microsoft.Extensions.Logging.LoggerFactory.Create%2A?displayProperty=nameWithType>.
+- When using DI without a host, configure in <xref:Microsoft.Extensions.DependencyInjection.LoggingServiceCollectionExtensions.AddLogging%2A?displayProperty=nameWithType>.
+- When using a host, configure with <xref:Microsoft.Extensions.Hosting.HostApplicationBuilder.Logging?displayProperty=nameWithType>, <xref:Microsoft.AspNetCore.Builder.WebApplicationBuilder.Logging?displayProperty=nameWithType> or other host specific APIs.
+
+This example shows setting the console [logging provider](logging-providers.md) and several [filters](#how-filtering-rules-are-applied).
+
+```csharp
+using Microsoft.Extensions.Logging;
+
+using var loggerFactory = LoggerFactory.Create(static builder =>
+{
+    builder
+        .AddFilter("Microsoft", LogLevel.Warning)
+        .AddFilter("System", LogLevel.Warning)
+        .AddFilter("LoggingConsoleApp.Program", LogLevel.Debug)
+        .AddConsole();
+});
+
+ILogger logger = loggerFactory.CreateLogger<Program>();
+logger.LogDebug("Hello {target}", "Everyone");
```

Comment:
```suggestion
logger.LogDebug("Hello {Target}", "Everyone");
```

Reply:
sorry I feel silly - after fixing ~5 other instances of that same capitalization issue I added a new one :( Thanks for catching it.

Reply:
No worries, all good. 🤓